pkg/virt-config/feature-gates.go

```diff
@@ -59,7 +59,8 @@ const (
 	DisableCustomSELinuxPolicy = "DisableCustomSELinuxPolicy"
 	// KubevirtSeccompProfile indicate that Kubevirt will install its custom profile and
 	// user can tell Kubevirt to use it
-	KubevirtSeccompProfile = "KubevirtSeccompProfile"
+	KubevirtSeccompProfile   = "KubevirtSeccompProfile"
+	HotplugNetworkIfacesGate = "HotplugNICs"
```

Comment:
Please express this FG in the commit message and clarify how it behaves from the client perspective (it ignores commands? It rejects them with an error?)